src/golang.org/exercises/jewels-and-stones/jewels-and-stones.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+)
+
+func numJewelsInStones(J string, S string) int {
+	jewels := make(map[string]struct{})
+	for i := range J {
+		jewels[string(J[i])] = struct{}{}
+	}
+
+	count := 0
+	for i := range S {
+		if _, ok := jewels[string(S[i])]; ok {
+			count++
+		}
+	}
+	fmt.Printf("Count: %d \n", count)
```

Comment:
I'd leave IO (printing) to the caller.  For cleanliness, computational or library functions typically should not call print statements directly.  Either let the caller print or use a log statement.

Reply:
moved printing outside of computational function in commit https://github.com/kcarrel/Golang-Training-Notes/pull/1/commits/0722bd650a57c4f9ee8e1a4801c15decc3142b5f